__tests__/components/card.spec.js

```diff
@@ -1,6 +1,83 @@
-import {loadFixture, testVM} from '../helpers';
+import { loadFixture, testVM } from '../helpers'
 
 describe('card', async() => {
-    beforeEach(loadFixture('card'));
-    testVM();
-});
\ No newline at end of file
+    beforeEach(loadFixture('card'))
+    testVM()
+
+    // since our class test depends on the card block,
+    // run this test first
+    it('all examples should contain card block', async() => {
+        const { app: { $refs, $el } } = window
+
+        const refs = ['simple_card', 'standard_card', 'img_card', 'img_overlay_card']
+
+        refs.forEach((ref) => {
+            const childNodes = [...$refs[ref].$el.childNodes]
+            const cardBlock = childNodes
+                .find(el => el.classList && el.classList.contains('card-block'))
+
+            expect(cardBlock).toBeDefined()
+        })
+    })
+
+    it('should contain class names', async() => {
+        const { app: { $refs, $el } } = window
+
+        expect($refs.simple_card).toHaveClass('card card-success card-inverse')
+        expect($refs.standard_card).toHaveClass('card')
+        expect($refs.img_card).toHaveClass('card')
+        expect($refs.img_overlay_card).toHaveClass('card card-inverse')
+
+        const blockEl = [...$refs.img_overlay_card.$el.childNodes]
+            .find(el => el.classList && el.classList.contains('card-block'))
+
+        expect(blockEl.classList.contains('card-img-overlay')).toBe(true)
+    })
+
+    it('should contain text content', async() => {
+        const { app: { $refs, $el } } = window
+
+        expect($refs.simple_card.$el.textContent).toContain('Simple Card')
+        expect($refs.standard_card.$el.textContent.trim()).toContain('Last updated 3 mins ago')
```

Comment:
Yeah, wasn't sure how strict the `toContain` equality check was. A lot of those nodes have new-lines, etc. Noted that it isn't necessary!